be/src/exec/hash_join_node.cpp

```diff
@@ -454,6 +460,23 @@ pipeline::OpFactories HashJoinNode::decompose_to_pipeline(pipeline::PipelineBuil
 
     size_t num_right_partitions = context->source_operator(rhs_operators)->degree_of_parallelism();
 
+    auto executor = std::make_shared<spill::IOTaskExecutor>(ExecEnv::GetInstance()->pipeline_sink_io_pool());
```

Comment:
it's recommended to use other thread pool. the tasks running in pipeline_sink_io_pool are submitted by brpc::execution_queue which only support batch execution, and these tasks usually need to access external system, the duration of a single task may not be controlled. spill tasks will be easily affected by other tasks if we put it here.

Reply:
how about using scan io thread pool ? it only contains local disk io task

Reply:
we could change it later.